packages/devtools_app/lib/src/primitives/feature_flags.dart

```diff
@@ -12,14 +12,20 @@
 ///   "args": [
 ///     "--dart-define=enable_experiments=true"
 ///   ]
-const bool _kEnableExperiments = bool.fromEnvironment('enable_experiments');
+const bool _kEnableExperiments = bool.fromEnvironment('enable_experiments') || _kEnableBeta;
+
+/// If true, features, ready for beta testing, will be on.
+///
+/// See [_kEnableExperiments] documentation  for usage.
+const bool _kEnableBeta =
+    bool.fromEnvironment('enable_beta');
```

Comment:
you can also add `!kReleaseMode || ...` so that when developing DevTools in debug or profile mode, this feature will be turned on by default, but it will not be turned on in release builds until we remove the beta flag